### PR TITLE
Add glob pattern support in storagetools

### PR DESCRIPTION
**Pull request by ReshetnikovPavel**
### Database name
Wal-g provides support for many databases, please write down name of database you uses.

# Pull request description
Add glob pattern support for storagetools ls, rm and cat commands

### Describe what this PR fix
// problem is ...

### Please provide steps to reproduce (if it's a bug)
// it can really help

### Please add config and wal-g stdout/stderr logs for debug purpose

also you can use WALG_LOG_LEVEL=DEVEL for logs collecting
<details><summary>If you can, provide logs</summary>
<p>
```bash
any logs here
```
</p>
</details>

### CODE REVIEW

File: internal/storagetools/remove_handler_test.go [hunk 1]
```diff
@@ -72,4 +72,38 @@ func TestHandleRemove(t *testing.T) {
 			assert.False(t, exists)
 		}
 	})
+
+	t.Run("remove files that match glob pattern", func(t *testing.T) {
```
Comment (debebantur) [inline internal/storagetools/remove_handler_test.go:76]: It would be great to add unit tests also for cat handler and list handler

**Comment by debebantur** [comment]
Also check failing tests, seems like there is a bug
`panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/wal-g/wal-g/cmd/common/st.glob..func6.1({0x3fb1aa0, 0xc0005b3dc0})
	/go/src/github.com/wal-g/wal-g/cmd/common/st/folder_list.go:25 +0x16b`

**Comment by ReshetnikovPavel** [comment]
> Also check failing tests, seems like there is a bug
> 
> ```
> panic: runtime error: index out of range [0] with length 0
> 
> goroutine 1 [running]:
> github.com/wal-g/wal-g/cmd/common/st.glob..func6.1({0x3fb1aa0, 0xc0005b3dc0})
> 	/go/src/github.com/wal-g/wal-g/cmd/common/st/folder_list.go:25 +0x16b
> ```

Oh, it was already there. Seems to be quite easy to fix though. Okay, I'll do it

File: internal/storagetools/remove_handler_test.go (hunk 1 above)
Comment (debebantur) [reply internal/storagetools/remove_handler_test.go:76]: Actually, it seems like another task

**Review by debebantur — APPROVED** [review]

**Pull request merged**